t/22_ipaddr2.t

```diff
@@ -940,4 +942,32 @@ subtest '[ipaddr2_refresh_repo]' => sub {
     ok((any { /zypper ref/ } @calls), 'Call zypper ref');
 };
 
+subtest '[get_private_ip_range]' => sub {
+    my %ip_range = sles4sap::ipaddr2::get_private_ip_range();
+    my %expected_value = (vnet_address_range => '192.168.0.0/16', subnet_address_range => '192.168.0.0/24', priv_ip_range => '192.168.0');
+    is_deeply \%ip_range, \%expected_value, "No worker_id, return 192.168.0.0 ip range";
+
+    set_var('WORKER_ID', '123');
```

Comment:
Since `WORKER_ID` is being set here for the unit test, I think it should be cleared (i.e. `set_var('WORKER_ID', undef)`) at the end of the unit test around L955.